src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +54,33 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService =
+                (MovieSessionService) injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService authenticationService =
+                (AuthenticationService) injector.getInstance(AuthenticationService.class);
+        User newUser = null;
+        try {
+            newUser = authenticationService.register("newuser@gmail.com", "pass");
+            System.out.println(newUser);
+        } catch (RegistrationException e) {
+            throw new RuntimeException("Can't register a user: " + newUser, e);
+        }
```

Comment:
in main we can rethrow exception to the method signature instead of try/catch but that is up to you